components/EmotionCard.js

```diff
@@ -52,15 +64,39 @@ const StyledLink = styled(Link)`
   }
 `;
 
-const StyledEmotionCard = styled.section`
-  display: flex;
-  flex-direction: column;
-  align-items: left;
-  font-size: 1rem;
+const StyledOuterBox = styled.div`
+  background-color: #e0e1f0;
+  border: 1px solid #d3d3d3;
+  border-radius: 0.5rem;
+  box-shadow: 0px 1px 4px #00000029;
```

Comment:
Just a suggestion: Use custom variables for colors and shadows to make future changes easier.